backend/base/templates/org_locationimages.html

```diff
@@ -198,6 +198,11 @@
                                                     <td>
                                                         <div class="update-icon">
                                                         <a href="{% url 'organization_deleteimage' pk=image.pk %}" class="btn btn-danger"><i class="fa-solid fa-trash"></i></a>
+                                                        <!-- <form method="post" action="{% url 'organization_deleteimage' pk=image.pk %}">
+                                                            {% csrf_token %}
+                                                            <input type="hidden" name="image_id" value="{{ image.id }}">
+                                                            <button type="submit" class="btn btn-danger"><i class="fa-solid fa-trash"></i></button>
+                                                        </form> -->

```

Comment:
comments can be removed

Reply:
That was partially written code for image delete in list itself instead of redirecting to another page.